web/src/helpers/redux.js

```diff
@@ -35,12 +35,10 @@ function initDataState() {
     Object.keys(TIME).forEach((agg) => {
       zone[TIME[agg]] = { details: [], overviews: [], isExpired: true };
     });
-
-    zone.config.capacity = zoneConfig.capacity;
```

Comment:
As far as I can tell we don't use this at all in the frontend and get all the capacities from the state files.